lib/ocpp/v201/smart_charging.cpp

```diff
@@ -322,12 +322,12 @@ SmartChargingHandler::validate_profile_schedules(ChargingProfile& profile,
 
         for (auto i = 0; i < schedule.chargingSchedulePeriod.size(); i++) {
             auto& charging_schedule_period = schedule.chargingSchedulePeriod[i];
-            // K01.FR.19
+            // K01.FR.48 and K01.FR.19
```

Comment:
`K01.FR.48` also has "AND the EVSE is NOT going to use the received phaseToUse". I don't see a check for that here. I assume this is implied by the false value in the device model, but it is not clear why the requirement has the AND in that case. The full text is:

"When a SetChargingProfileRequest with a value for phaseToUse is received AND the EVSE is NOT capable of switching the phase connected to the EV, which is indicated by ACPhaseSwitchingSupported not being implemented or defined as false AND the EVSE is NOT going to use the received phaseToUse"

So if the `ACPhaseSwitchingSupported` already implies that the EVSE is not using it, then why does the spec need the last AND?

Reply:
This was brought up last month at the Cloud Comms WG, where it was concluded that that part of the predicate in question is redundant.

Reply:
Please add a link to this discussion (mailing list/zulip chat summary) for the record.

Reply:
Issue created per the discussion with the community - https://github.com/EVerest/libocpp/issues/723